source/_static/js/custom.js

```diff
@@ -0,0 +1,31 @@
+var app = {
+
+    check_logo_height: function() {
+        var logo = $('div[itemprop="articleBody"] > .sidebar')
+        var hr = $('div[itemprop="articleBody"] > .section > hr')
+        var logo_end = 0
+        var hr_position = 0
+        if (logo.length) {
+            logo_end = logo.offset().top + logo.outerHeight();
+        }
+        if (hr.length) {
+            hr_position = hr.offset().top;
+        }
+        console.log('[logo] ' + logo_end + " → " + hr_position);
+        if (logo_end > hr_position) {
+            var margin = parseInt(hr.css('margin-bottom'), 10)
+            var offset = Math.ceil(logo_end - hr_position)
+            var new_margin = margin + offset
+            console.log( '[logo] pushing first break down to ' + margin + ' + ' + offset + ' → ' + new_margin);
+            hr.css('margin-bottom', new_margin + 'px');
+        }
+    },
+
+    inti: function() {
```

Comment:
init?

Reply:
I put the `check_logo_height` function in an `app` object… to preserve some global namespace. If needed, new functions and other stuff can be added to the `app` object.  Calling `app.init()` does all the stuff that should happens automatically on each page load.